Visual markers are not visible in v2+ in VSCode

After updating to version v2+, the visual markers no longer appear. I tested both versions 2.0.2 and 2.1.1 with the same result.

Reports are working correctly, but the visual markers were very helpful.

Reverting to version 1.3.4 restores the visual markers and everything works as expected.

Environment:

Visual Studio Code: 1.110.1

@blazejkustra on v2 I get this:
```
[2026-03-25T15:44:23.436Z] CLIENT LOG: react-compiler-marker is being activated!
[2026-03-25T15:44:23.440Z] CLIENT LOG: React Compiler Marker ✨: Commands registered.
[2026-03-25T15:44:23.441Z] CLIENT LOG: React Compiler Marker ✨: Initialization complete.
[2026-03-25T15:44:23.722Z] SERVER LOG: Client connected: Visual Studio Code 1.112.0 (tooltipFormat: markdown, hover: disabled)
[2026-03-25T15:44:24.437Z] CLIENT LOG: React Compiler Marker LSP client started
[2026-03-25T15:44:24.435Z] SERVER LOG: React Compiler Marker LSP Server initialized
[2026-03-25T15:44:25.021Z] SERVER ERROR: Failed to load babel-plugin-react-compiler from workspace: Cannot find module '\c%3A\repos\WorkZone\WZONE\node_modules\babel-plugin-react-compiler'
```

But I have `babel-plugin-react-compiler` installed:
```
C:\repos\WorkZone\WZONE>npm list babel-plugin-react-compiler
@wzone/source@2026.0.4 C:\repos\WorkZone\WZONE
├── babel-plugin-react-compiler@1.0.0
└─┬ next@16.2.1
  └── babel-plugin-react-compiler@1.0.0 deduped
```

I'm on Windows, Node v24.14.0

@creage I don't have access to a Windows machine, maybe you would like to try to fix it yourself?

I feel like this is the main clue: "SERVER ERROR: Failed to load babel-plugin-react-compiler from workspace: Cannot find module '\c%3A\repos\WorkZone\WZONE\node_modules\babel-plugin-react-compiler'"

--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@
 
 | IDE  | Installation |
 |------|--------------|
-| VS Code / Cursor | [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=blazejkustra.react-compiler-marker) \| [Open VSX](https://open-vsx.org/extension/blazejkustra/react-compiler-marker) |
+| VS Code / Cursor / Antigravity | [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=blazejkustra.react-compiler-marker) \| [Open VSX](https://open-vsx.org/extension/blazejkustra/react-compiler-marker) |
 | WebStorm / IntelliJ IDEA | [IntelliJ marketplace](https://plugins.jetbrains.com/plugin/29540-react-compiler-marker) |
 | Neovim | [Setup instructions](packages/nvim-client/README.md) |
 | Zed (alpha) | [Setup instructions](packages/zed-client/README.md) |
@@ -68,7 +68,7 @@ npx react-compiler-marker --format json . > report.json
 
 See the [CLI README](packages/cli/README.md) for all options.
 
-### VS Code / Cursor
+### VS Code / Cursor / Antigravity
 
 1. Install from [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=blazejkustra.react-compiler-marker) or search "React Compiler Marker" in Extensions
 3. Open a React component file - markers appear automatically

--- a/packages/server/src/server.ts
+++ b/packages/server/src/server.ts
@@ -13,6 +13,7 @@ import {
 } from "vscode-languageserver/node";
 
 import { TextDocument } from "vscode-languageserver-textdocument";
+import { fileURLToPath } from "node:url";
 import {
   checkReactCompiler,
   getCompiledOutput,
@@ -78,9 +79,11 @@ function logError(error: string): void {
 }
 
 connection.onInitialize((params: InitializeParams): InitializeResult => {
-  workspaceFolder = params.workspaceFolders?.[0]?.uri;
-  if (workspaceFolder?.startsWith("file://")) {
-    workspaceFolder = workspaceFolder.slice(7);
+  const workspaceFolderUri = params.workspaceFolders?.[0]?.uri;
+  if (workspaceFolderUri?.startsWith("file://")) {
+    workspaceFolder = fileURLToPath(workspaceFolderUri);
+  } else {
+    workspaceFolder = workspaceFolderUri;
   }
 
   // Store client name for feature detection

